src/input.c

```diff
@@ -24,7 +24,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include <wpe/input.h>
+#include "../include/wpe/input.h"
```

Comment:
This include looks weird to me, can't we keep the include? "wpe/input.h" would still be a local header, no?

Reply:
Why is it weird? It's nothing else than the relative path to `input.h` from `src/input.c` :thinking: — which guarantees that the local version of the header from the source tree is only ever used because the compiler will always find it, therefore always bypassing the search through paths specified with `-I` and `-isystem` (so: less potential footguns); also it makes it unnecessary to pass `-I<srcdir>/include` to the compiler.

I can change it to `"wpe/input.h"` but I would rather leave it with the actual relative path.

(And, while not a concern here because libwpe is small, this practice makes compilation faster because it avoids cascades of `stat` syscalls through the header include directories :wink:)